src/common-models/schedule-error.model.ts

```diff
@@ -14,10 +14,32 @@ export enum ParseErrorCode {
   UNKNOWN_VALUE = "unknownSymbol",
 }
 
+export enum InputFileErrorCode {
+  EMPTY_FILE = "emptyFile",
+  NO_CARETAKER_SECTION = "noCaretakerSection",
```

Comment:
Wszędzie w aplikacji tłumaczymy słowo opiekunka jako `babysitter`, więc może mogłabyś to zmienić, żeby nie wprowadzać zamieszania